pkg/products/obo/prometheusRules.go

```diff
@@ -304,21 +304,21 @@ func OboAlertsReconciler(logger l.Logger, installation *integreatlyv1alpha1.RHMI
 					Alert: fmt.Sprintf("%sIsInReconcilingErrorState", strings.ToUpper(installationName)),
 					Annotations: map[string]string{
 						"sop_url": resources.SopUrlRHOAMIsInReconcilingErrorState,
-						"message": fmt.Sprintf("%s operator has finished installing, but has been in a error state while reconciling for last 10 minutes", strings.ToUpper(installationName)),
+						"message": fmt.Sprintf("%s operator has finished installing, but has been in a error state while reconciling for last 1 hour", strings.ToUpper(installationName)),
 					},
 					Expr:   intstr.FromString(fmt.Sprintf(`(%s_status{stage!="complete"} > 0) * on(pod) group_left(to_version, version) (%[1]s_version{to_version="",version=~".+"} > 0)`, installationName)),
-					For:    "10m",
-					Labels: map[string]string{"severity": "warning", "product": installationName},
+					For:    "60m",
+					Labels: map[string]string{"severity": "critical", "product": installationName},
 				},
 				{
 					Alert: fmt.Sprintf("%sInstallationControllerReconcileDelayed", strings.ToUpper(installationName)),
 					Annotations: map[string]string{
 						"sop_url": resources.SopUrlAlertsAndTroubleshooting,
-						"message": fmt.Sprintf("The reconcile function of the installation controller in a completed state of %s operator is taking more than 30 minutes to complete", strings.ToUpper(installationName)),
+						"message": fmt.Sprintf("The reconcile function of the installation controller in a completed state of %s operator is taking more than 2 hours to complete", strings.ToUpper(installationName)),
 					},
 					Expr:   intstr.FromString(installationName + `_version{version=~".+", to_version=""} * on(pod) (installation_controller_reconcile_delayed == 1)`),
-					For:    "1m",
-					Labels: map[string]string{"severity": "warning", "product": installationName},
+					For:    "90m",
```

Comment:
is it correct? Installation delay 90 min? Seems to me it can be around 10-15 min Critical

Reply:
we discussed. Thanks.